.github/workflows/release.yml

```diff
@@ -64,21 +69,26 @@ jobs:
           echo The branch to release from:         ${{ inputs.target_branch }}
           echo The release version number:         ${{ inputs.version_number }}
           echo Build script path:                  ${{ inputs.build_script_path }}
+          echo Environment setup script path:      ${{ inputs.env_setup_script_path }}
           echo AWS S3 bucket name:                 ${{ inputs.s3_bucket_name }}
           echo Package test command:               ${{ inputs.package_test_command }}
           echo Test run:                           ${{ inputs.test_run }}
 
   bump-version-generate-changelog:
```

Comment:
It's a bit confusing that the purpose of this job for this spark repo is currently just to audit that the version bump and changelog generations have been done based solely off the fact that `env_setup_script_path` is blank.

Please add a comment to that effect and also describing that the version_bump.yml needs to be run manually for this repo and the PR must be merged into before running this release step.

Reply:
I provided the following updates:
- Updated workflow docs to include info about `version_bump.yml`;
- Made the `env_setup_script_path` input optional;
- Rename `bump-version-generate-changelog` to `audit-version-and-changelog` to make more sense;
- Add comments for `audit-version-and-changelog` stage to provide more context;